cabal.project.freeze

```diff
@@ -0,0 +1,2528 @@
+constraints: any.AC-Angle ==1.0,
```

Comment:
I am not familiar with the purpose of this file, but it looks like version pinning of all direct and indirect dependencies. How do we bump these versions? Is there a tool that does it?

Reply:
The freeze file has been created by the `stack2cabal` tool. We don't know yet how easy it is to upgrade without stackage yet. This PR doesn't commit it us to using cabal however.